.contributing/the-process.md

```diff
@@ -125,4 +154,24 @@ Notes on the various tools 
 
 -   Google docs are great for creating longer, more complex documents (eg. with diagrams and a TOC)
 
--   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
\ No newline at end of file
+-   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
+
+
+Key
+====
+
+- "IS" : input session
+- "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* - User Flow files - *Section* (CL)"
+- "input task": 
+  - Create a task in Asana with title "IS: Lofi Sketch < x. Use Case" and:
+      - Add a link to the google drawing in the task description
+      - Move the task to the "Blockers & Reviews" column
+      - Add a due date of tomorrow
+      - At standdown:
+        - resolve any conversations (add RTCs for the next day if need be)
+      - At next standup:
+        - remind everyone the doc will be sealed at end of day
+      - At next standdown:
+        - give everyone a final warning
+                resolve any conversations (use real time chat if need be)
```

Comment:
is this missing a '-' or meant to have the large indentation?

Reply:
missing the '-', good pickup!